### PR TITLE
Update changelog and version

### PR DESCRIPTION
**Pull request by Yuav**
Fixed issue with "puppet module install yum" unable to satisfy dependencies

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 2016-09-26 - Release 0.9.15
+
+- Remove invalid dependency from metadata
```
Comment (bastelfreak) [inline CHANGELOG.md]: please add the other changes as well. You can find a list here: https://github.com/voxpupuli/puppet-yum/compare/v0.9.14...master

File: metadata.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-yum",
-  "version": "0.9.15-rc0",
+  "version": "0.9.15",
   "author": "Voxpupuli",
```
Comment (bastelfreak) [inline metadata.json]: Please update to "Vox Pupuli"

Comment (Yuav) [inline metadata.json]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (Yuav) [inline CHANGELOG.md]: Fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+- Manage yum::versionlock with concat
+- Puppet no longer creates empty files when using Hiera and create_resources()
+  to create Yum keys
+
 ## 2016-08-15 - Release Releasing 0.9.14
```
Comment (alexjfisher) [inline CHANGELOG.md]: I may have taken the opportunity to fix the 2 instances of 'Release Releasing'

Comment (Yuav) [inline CHANGELOG.md]: Fixed

**Pull request merged**